PeerioTechnologies/peerio-desktop
# fix(sidebar): sidebar has border only when opened

#### Relevant info and issue/PR links 
 
https://app.clubhouse.io/peerio/story/1505/when-chat-sidebar-is-closed-we-can-see-its-border

#### Testing instructions  

Close sidebar, see no border on the right. Open sidebar, see the border to the left of sidebar.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
